model_analyzer/analyzer.py

```diff
@@ -98,11 +105,22 @@ def profile(self, client, gpus):
 
         self._profile_models()
 
+        if (self._state_manager.exiting()):
+            sys.exit(1)
+
+        # TODO: TMA-792: This won't be needed once the Results class is used in profile
+        self._analyze_models()
+
+        self._create_summary_tables(verbose)
+
+        # TODO: TMA-790: Skip this step if the CLI option indicates the user doesn't want summary reports
```

Comment:
I still think profile-only should also skip the csv output as well (which I think is 2 lines above this?)

Reply:
Done